src/index.js

```diff
@@ -45,11 +42,8 @@ module.exports = (config) => {
     ...config,
   };
 
-  if (configuration.transport !== utils.emptyLogger) {
-    process.emitWarning(
-      "The 'transport' configuration option is deprecated. Please use the 'logger' option instead.",
-      'DeprecationWarning'
-    );
+  if (configuration.transport) {
+    throw new Error('Using removed deprecated "transport" option. Use the "logger" option instead.');
```

Comment:
Now throwing error if the transport is used instead of writing a deprecation warning.